rest/dep_workflow.go

```diff
@@ -61,7 +64,21 @@ func (s *Server) newWorkflowHandler(w http.ResponseWriter, r *http.Request) {
 	} else {
 		data["continueOnError"] = strconv.FormatBool(false)
 	}
-
+	// Get instances selection if provided in the request body
+	body, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		log.Panic(err)
+	}
+	var wfRequest WorkflowRequest
+	if err = json.Unmarshal(body, &wfRequest); err != nil {
```

Comment:
Why not checking if body content is not empty before json unmarshall ?

Reply:
Took example on dep_custom.go 
It exists a function to check that a request has an empty body ?